pkg/apis/serving/v1alpha1/kfservice_framework_scikit.go

```diff
@@ -17,17 +17,41 @@ import (
 	v1 "k8s.io/api/core/v1"
 )
 
-func (s *ScikitLearnSpec) CreateModelServingContainer(modelName string) *v1.Container {
-	// TODO: https://github.com/kubeflow/kfserving/issues/58
+const (
+	SKLearnEntrypointCommand = "python"
+	SKLearnServingGRPCPort   = "9000"
+	SKLearnServingRestPort   = "8080"
+	SKLearnServingImageName  = "animeshsingh/sklearnserver"
+
+	DefaultSKLearnServingVersion = "latest"
+)
+
+func (s *SKLearnSpec) CreateModelServingContainer(modelName string) *v1.Container {
+	//TODO(@animeshsingh) add configmap for image, default resources, readiness/liveness probe
 	return &v1.Container{
-		Image: "notimplementedyet",
+		Image:     SKLearnServingImageName + ":" + s.RuntimeVersion,
+		Command:   []string{SKLearnEntrypointCommand},
+		Resources: s.Resources,
+		Args: []string{
+			// TODO: Allow setting rest and grpc ports @animeshsingh
+			// "--port=" + SKLearnServingGRPCPort,
+			// "--rest_api_port=" + SKLearnServingRestPort,
+			"-m",
+			"sklearnserver",
```

Comment:
We shouldn't need to pass in -m sklearnserver. Let's just setup the dockerfile with this and have it ready to accept --model_name and --model_dir

Reply:
Done